src/libraries/Common/src/System/Security/Cryptography/Asn1/AsnXml.targets

```diff
@@ -29,13 +29,13 @@
     <Exec
       IgnoreExitCode="true"
       StandardOutputImportance="Low"
-      Command="$(_AsnXmlDiffCmd) @(AsnXml -> '$(_AsnIntermediatePath)%(filename).cs') @(AsnXml -> '%(Identity).cs')">
+      Command="$(_AsnXmlDiffCmd) @(AsnXml -> '&quot;$(_AsnIntermediatePath)%(filename).cs&quot;') @(AsnXml -> '&quot;%(Identity).cs&quot;')">
       <Output TaskParameter="ExitCode" ItemName="_AsnXmlDiffCode" />
     </Exec>
 
     <Copy
       Condition="'@(_AsnXmlDiffCode)' != '0'"
-      SourceFiles="@(AsnXml -> '$(_AsnIntermediatePath)%(filename).cs')"
+      SourceFiles="@(AsnXml -> $(_AsnIntermediatePath)%(filename).cs')"
```

Comment:
Is this correct?

Reply:
looks like a ' was removed by accident?

Reply:
Doh, I left it out first, but then overlooked it when adding other changes.